rpi-update

```diff
@@ -316,8 +313,8 @@ function show_notice {
 	if ${NOTICE_HASH_EXISTS}; then
 		local NOTICE=$(echo "${FULL_NOTICE}" | tail -n+2)
 		local NEW_HASH=${FW_REV}
-		local LOCAL_lt_NOTICE=$(compare_hashes "${LOCAL_HASH}" lt "${NOTICE_HASH}")
-		local NEW_ge_NOTICE=$(compare_hashes "${NEW_HASH}" ge "${NOTICE_HASH}")
+		local LOCAL_lt_NOTICE=$(compare_hashes ${REPO_API_URI} "${LOCAL_HASH}" lt "${NOTICE_HASH}")
+		local NEW_ge_NOTICE=$(compare_hashes ${REPO_API_URI} "${NEW_HASH}" ge "${NOTICE_HASH}")
```

Comment:
quote these two for consistency

Reply:
Ok. There seems to be a mixture of styles for quoting elsewhere for REPO_API_URI but I think quotes for safety